### DISCUSSION TITLE
authentication required err, even after supplied HTTP token

### DISCUSSION THREAD

**Discussion by vijaymailb** [fluxcd/flux2#2353] [Q&A] [answered]
Hello,

I am getting below err while bootstrapping bibucket repo after supplying HTTP token. Can someone please suggest me is there any other credentials that need to supply on flux bootstapping?

Please enter your Bitbucket personal access token (PAT):
► connecting to src.private.zooplus.net
► cloning branch "feature/PSOD-1638_FluxCD" from Git repository "https://src.private.zooplus.net/scm/K8S/kops.git"
✗ failed to clone repository: authentication required

**Comment by kingdonb** [comment]
I think the Bitbucket bootstrap feature is new! I'm not sure how many people are using it. I'd like to follow up and try this out, to be sure if you're having a unique issue or if it's broken more broadly. (If anyone who is using Bitbucket already has seen this issue, it will be quicker if you're able to respond...)

**Reply by kingdonb** [reply]
We haven't seen any reply, I'm going to mark this as answered for now. Please let us know if you're still struggling with this issue @vijaymailb

**Comment by souleb** [comment, marked as answer]
can you provide the full command that you have entered?
